### PR TITLE
[PRED-4335] Script deprecation

### PR DESCRIPTION
**Pull request by dotkas**
# BATCH SCORING PULL REQUEST

This is a pull request into a **public** repository for Batch Scoring script maintained by DataRobot.

## RATIONALE

We are officially deprecating this script in favor of [the Batch Prediction CLI script](https://app.datarobot.com/docs/predictions/batch/cli-scripts.html).

## CHANGES

* Added a deprecation warning message various places

# Archive the repo

Could a GitHub admin please archive this repo to make it read-only?

### CODE REVIEW

File: datarobot_batch_scoring/consts.py [hunk 1]
```diff
@@ -6,6 +6,22 @@
...
+    'instead: '
+    '({underline}https://app.datarobot.com/docs/predictions'
+    '/batch/cli-scripts.html{reset}).'
+    .format(
+        yellow='\033[93m',
+        bold='\033[1m',
+        reset='\033[0m',
+        underline='\033[4m'
+    )
+   )
```
Comment (andrii-rubtsov) [inline datarobot_batch_scoring/consts.py:23]: :+1:

**Review by andrii-rubtsov — DISMISSED** [review]

**Comment by andrii-rubtsov** [comment]
Wondering if we should mark the repo as archived?

**Comment by dotkas** [comment]
@andrii-rubtsov yes! I also wrote that in the PR description. Can you do it? I don't have access.

**Comment by andrii-rubtsov** [comment]
> @andrii-rubtsov yes! I also wrote that in the PR description. Can you do it? I don't have access.

No, I do not have access either. Let us ask in #github-admins once the PR is merged.

**Review by andrii-rubtsov — APPROVED** [review]

**Pull request merged**